canonical/canonical.com
# Wd 3735 dev canonical com blog

## Done

Rebuild the blog landing page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check jinja logic, check against the design. 
- - Check if all unused local styles and components (cards for example) have been removed.

## Issue / Card

Fixes #

## Screenshots

![image](https://github.com/canonical/canonical.com/assets/2741678/dd4bd1d3-377d-424f-814f-4cb8fb2d0459)
